web-registry/src/components/organisms/MediaForm.tsx

```diff
@@ -162,13 +156,16 @@ const MediaForm: React.FC<MediaFormProps> = ({ submit, initialValues }) => {
             const report = await validate(
               graphData.shaclGraphByUri.graph,
               projectPageData,
-              'regen:ProjectPageMediaGroup',
+              'http://regen.network/ProjectPageMediaGroup',
             );
             for (const result of report.results) {
               const path: string | undefined = result.path?.value;
               if (path) {
-                if (path === 'regen:previewPhoto') {
-                  errors[path] = { '@value': requiredMessage };
+                const compactedPath = getCompactedPath(path) as
+                  | keyof MediaValuesErrors
+                  | undefined;
+                if (compactedPath === 'regen:previewPhoto') {
+                  errors[compactedPath] = { '@value': requiredMessage };
```

Comment:
@haveanicedavid I've added this `getCompactedPath` function so that it gets the corresponding "compacted" path, eg for `http://regen.network/previewPhoto` it would return `regen:previewPhoto` which is the formik field name